rerun_py/docs/package/index.md

```diff
@@ -1,8 +1,8 @@
 # Full Package
 
 - [rerun][]: Top-level User-facing APIs. See also: [Common APIs](../common/)
-    - [rerun.log][]: APIs for logging data
-    - [rerun.color_conversion][]: Conversion utilities related to colors
-    - [rerun.components][]: Helpers for constructing arrow components
+    - [rerun.archetypes][]: Common objects for logging primitives understood by the viewer, e.g. Points and Lines.
```

Comment:
```suggestion
    - [rerun.archetypes][]: Common objects for logging tagged data understood by the viewer, e.g. Points and Lines.
```

maybe better? I'm getting more and more uneasy around "primitives". Putting `tagged` here might be too much though (also we never call the indicators 'tagged'), but maybe something like this